src/vs/platform/extensionManagement/common/extensionGalleryService.ts

```diff
@@ -608,6 +608,11 @@ export class ExtensionGalleryService implements IExtensionGalleryService {
 					});
 				}
 			}
+			// ADS doesn't not support extension tags, we need to return empty array to avoid breaking some scenarios. e.g. file extension based recommendations.
```

Comment:
does this not need {{SQL CARBON EDIT}} because it's there for the whole function?

Reply:
this is already inside the {{SQL CARBON EDIT}}